docs/reference/release-policy.md

```diff
@@ -3,72 +3,110 @@ id: release-policy
 title: "Release policy"
 ---
 
-Camunda 8 follows the [Camunda release policy](https://camunda.com/release-policy/) with some specific clarifications which can be found below.
+Camunda 8 follows the [Camunda release policy](https://camunda.com/release-policy/) with the following specific clarifications.
 
-:::note
+:::info
+You can find deprecation and support announcements on the [Announcements](announcements.md) page.
+:::
+
+![Stable and alpha channels when provisioning a cluster](./img/diagram-releases.png)
+
+## Alpha features and releases
+
+It is important to understand the different ways the term "alpha" is used in the context of Camunda releases and features.
 
-Interested in deprecation and support announcements? Read more on the [Announcements](announcements.md) page.
+### Alpha feature
 
+Refers to a feature or component released as an alpha version, in an early state for you to test and participate in development by sharing your feedback before the feature reaches [general availability (GA)](/docs/reference/alpha-features.md#general-availability-ga). You can choose whether to enable alpha features in your cluster. See [alpha features](/docs/reference/alpha-features.md).
```

Comment:
Not all alpha features have to be toggled specifically.

```suggestion
Refers to a feature or component released as an alpha version, in an early state for you to test and participate in development by sharing your feedback before the feature reaches [general availability (GA)](/docs/reference/alpha-features.md#general-availability-ga). Some alpha features can / have to be toggled on for your cluster. See [alpha features](/docs/reference/alpha-features.md).
```

Reply:
Changed to: "Some alpha features require turning on for your cluster before you can use them."